packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -954,6 +978,11 @@ function parseHookName(functionName: void | string): string {
   } else {
     startIndex += 1;
   }
+
+  if (functionName.slice(startIndex).startsWith('unstable_')) {
+    startIndex += 'unstable_'.length;
+  }
```

Comment:
Never considered that. I guess we also have a similar problem with `experimental_useEffectEvent`.

Reply:
@jackpope @gsathya Do we have plans for using `experimental_useEffectEvent` soon? Will it be transpiled by the compiler?

I can put up a separate PR for it.

Reply:
Not that I know of. But it might be worth handling the pattern anyway. I think we can assume we'll create more `unstable_` or `experimental_` prefixes in the future, beyond `useEffectEvent` / `useContextWithBailout`.